Headers/DebugServer2/GDBRemote/DummySessionDelegateImpl.h

```diff
@@ -12,13 +12,15 @@
 #define __DebugServer2_GDBRemote_DummySessionDelegateImpl_h
 
 #include "DebugServer2/GDBRemote/SessionDelegate.h"
+#include "DebugServer2/Host/FileIO.h"
 
 namespace ds2 {
 namespace GDBRemote {
 
 class DummySessionDelegateImpl : public SessionDelegate {
 protected:
   bool _secure;
+  Host::FileIO _fileIO;
```

Comment:
I don't think this is good. You can have multiple `open` calls followed by multiple `close` calls. You should have an `std::list<FileIO>` or something like that.